pom.xml

```diff
@@ -33,17 +33,17 @@
         <dice-provider-artifact-id>dice-provider</dice-provider-artifact-id>
         <x509-provider-artifact-id>x509-provider</x509-provider-artifact-id>
 
-        <iot-device-client-version>1.32.1</iot-device-client-version>
-        <iot-service-client-version>1.33.0</iot-service-client-version>
-        <iot-deps-version>0.15.0</iot-deps-version>
-        <provisioning-device-client-version>1.10.0</provisioning-device-client-version>
-        <provisioning-service-client-version>1.9.0</provisioning-service-client-version>
-        <security-provider-version>1.5.0</security-provider-version>
-        <tpm-provider-emulator-version>1.1.2</tpm-provider-emulator-version>
+        <iot-device-client-version>2.0.0-preview-001</iot-device-client-version>
+        <iot-service-client-version>2.0.0-preview-001</iot-service-client-version>
+        <iot-deps-version>2.0.0-preview-001</iot-deps-version>
+        <provisioning-device-client-version>2.0.0-preview-001</provisioning-device-client-version>
+        <provisioning-service-client-version>2.0.0-preview-001</provisioning-service-client-version>
+        <security-provider-version>2.0.0-preview-001</security-provider-version>
+        <tpm-provider-emulator-version>undefined</tpm-provider-emulator-version>
         <tpm-provider-version>1.1.3</tpm-provider-version>
```

Comment:
this lonely fellow got no updates

Reply:
That is odd. I'll look into why this wasn't bumped, too